src/book/02-system/01-foundations/01-overview.mdx

```diff
@@ -22,3 +22,42 @@ After motions is hardware IO. The hardware will output the commands and input th
+| `filter_playback`    | Filter sensor data, data playback and NUsight                                                         | Use for playing back raw sensor data and viewing the filtered data in NUsight.                                                                                                                                                                                          |
+| `firmwareinstaller`  | Firmware installer                                                                                    | Used to install firmware to the robot.                                                                                                                                                                                                                                  |
+| `gamecontrollertest` | Real robot sensors, game controller and NUsight                                                       | Used for testing the connection between the robot and [GameController](https://github.com/RoboCup-Humanoid-TC/GameController).                                                                                                                                          |
+| `gazebo`             | Gazebo robot sensors, motion modules and NUsight                                                      | Used to run motions with a robot in [Gazebo](http://gazebosim.org/).                                                                                                                                                                                                    |
+| `headbehtest`        | Real robot sensors and camera, vision modules, script modules, head controller and behaviour          | Used to test head behaviour, particularly with balls and goals.                                                                                                                                                                                              |
+| `keyboardwalk`       | Real robot sensors, motions, keyboard walk and NUsight                                                | Used for demonstrating the robot manually with a keyboard, particularly in a setting without a field.                                                                                                                                                                   |
+| `keyboardwalkfake`   | As above but with a basic simulated robot (not webots)                                                | Used for testing the code without a real robot, potentially to make sure there are no runtime errors. Used to demonstrate basic functionality of the robot. Part of the [Getting Started](/guides/main/getting-started) example since it does not require a real robot. |
```

Comment:
```suggestion
| `keyboardwalkfake`   | As above but with a basic simulated robot (not webots)                                                | Used for testing the code for run-time issues without a real robot. Used to demonstrate basic functionality of the robot. Part of the [Getting Started](/guides/main/getting-started) example since it does not require a real robot. |
```